cfpb/cfpb.github.io
# Remove articles, fix 'About' footer link

**Taken from issue #284: JIRA 5240: Remove the blog from github.cfpb.io 

_Remove the blog

~~Remove https://cfpb.github.io/articles/~~
~~Remove all https://cfpb.github.io/articles/ child pages~~
Remove/Hide link to cfpb.github.io/articles
Remove the "Learn more" to the blog from https://cfpb.github.io/_

**To test:**
Visit http://localhost:4000/ and verify "Learn more" link has been removed.

Hover over "About" page in footer in footer and verify URL change (/cfpb-open-tech)

- Once at the "About" page, hover over the "CFPB Open Tech" logo/text in upper left corner and verify URL is set back to the landing page. (cfpb.github.io)

~~Navigate to cfpb.github.io/articles and verify page has been removed.~~
cfpb.github.io/articles will still be live, but without linkage on any page that will take the user to 'articles'.
